test/Microsoft.IdentityModel.Xml.Tests/DSigSerializerTests.cs

```diff
@@ -45,17 +45,23 @@ public class DSigSerializerTests
         [Fact]
         public void GetSets()
         {
-            var dsigSerializer = new DSigSerializer();
+
+            var serializer = new DSigSerializer();
```

Comment:
nit: looks like the serializer is not needed here. Extra new line.

Reply:
Thanks, some weird spacing in that file also.